README.md

```diff
@@ -7,36 +7,53 @@ Deployed via [Netlify](https://www.netlify.com/)
 
 ## Development
 
-This is a [Jekyll][jekyll] static site. We include a [Devcontainer][devcontainer] configuration to help with local testing
+This is a [Jekyll][jekyll] static site. We include a [Dev Container][devcontainer] configuration to help with local testing
 and development.
 
 1. Open the repository directory in VS Code
 1. `Ctrl/Cmd+Shift+P` to bring up the Command Palette
-1. Enter `Remote-Containers` to filter the command list
+1. Start typing `devcontainers` to filter the command list
 1. Enter or select `Rebuild and Reopen in Container` to start from scratch
 1. Enter or select `Reopen in Container` to reopen the last devcontainer used
 
 Once inside the devcontainer, you have the manually start the site:
 
 1. `Ctrl/Cmd+Shift+P` to bring up the Command Palette
 1. Enter `Tasks: Run Task`
-1. Enter or select `Jekyll: Build Dev`
+1. Enter or select `Jekyll: Build and Serve Locally`
 
-The site is running on <http://127.0.0.1>; check the VS Code _Ports_ tab for the exact port. Auto rebuild/reload will be active
-and will watch the site files for changes.
+The site is now running on <http://127.0.0.1>; check the VS Code _Ports_ tab for the exact port. You may also see a notification appear indicating the port and prompting you to open it in your browser. Auto rebuild/reload will be active and will watch the site files for changes.
 
 ## Analytics
 
 Site analytics is tracked by Google Analytics, version 4. Ask an administrator to grant you access.
 
-## Resource Requests
+## Resources and Press content routing
 
-We have a semi-automated way of handling resource requests via a Python script combined with the [`gh`](http://cli.github.com/) command.
+The Resources and Press sections are each a collection of items that may either (a) render their content on a cal-itp.org page or (b) reference an different URL.
```

Comment:
the canonical domain for this site is `calitp.org` -- `cal-itp.org` is just a redirect/secondary domain.